lms/djangoapps/verify_student/views.py

```diff
@@ -1154,10 +1154,22 @@ def results_callback(request):
     except SoftwareSecurePhotoVerification.DoesNotExist:
         log.error("Software Secure posted back for receipt_id %s, but not found", receipt_id)
         return HttpResponseBadRequest("edX ID {} not found".format(receipt_id))
+    context = {'user': attempt.user}
```

Comment:
nit: you don't need to pass the entire user context, since you are only using email and user_id